rollbar-java/src/main/java/com/rollbar/notifier/sender/queue/DiskQueue.java

```diff
@@ -78,11 +81,13 @@ public Payload peek() {
   }
 
   private void writeToFile(Payload payload) {
+    ObjectOutputStream objectOut = null;
     File file = new File(queueFolder.getAbsolutePath(), createFilename(payload));
-    try (FileOutputStream fileOut = new FileOutputStream(file);
-        ObjectOutputStream objectOut = new ObjectOutputStream(fileOut)) {
+    try {
+      objectOut = new ObjectOutputStream(new FileOutputStream(file));
       objectOut.writeObject(payload);
     } catch (Exception e) {
+      ObjectsUtils.close(objectOut);
```

Comment:
Could you please move it to a `finally` block?

Reply:
Ok.